release-team/release-team-selection.md

```diff
@@ -59,6 +59,8 @@ _The new Release Team Lead can be selected via lazy consensus of the current Rel
 
 Same as [Release Team Lead](#release-team-lead).
 
+The release lead should reach out to all release lead shadows and release team leads directly from the previous cycle to gather input from people interested in helping shadowing the new release.
```

Comment:
When we say "should reach out" this is not a must but make an attempt? Is my understanding correct or the wording?

Reply:
Your understanding is correct. Its "highly suggested". We could also change this to "Has to" if we want to make this mandatory (but in that case, who's is going to check that?).

Reply:
No, not looking to make it mandatory. Thanks for the clarification.